src/footer/footer.yaml

```diff
@@ -0,0 +1,145 @@
+accessibilityCriteria: |
+  Text and Links in the component must:
+  - have a text contrast ratio higher than 4.5:1 against the background colour to meet [WCAG AA](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast)
+
+  Links in the component must:
+  - accept focus
+  - be focusable with a keyboard
+  - be usable with a keyboard
+  - indicate when they have focus
+  - change in appearance when touched (in the touch-down state)
+  - change in appearance when hovered
+  - have visible text
+
+  Images in the component:
+  - Open Government Licence (OGL) icon must be presentational and with a accompanying text including a link describing OGL.
+  - crest icon should have an accessible name that explains what it is.
+
+  Landmarks and Roles:
+  - avoid navigation landmarks or roles
+    "The footer element alone is sufficient for such cases; while a nav element can be used in such cases, it is usually unnecessary." - (https://www.w3.org/TR/html52/sections.html#the-nav-element)
+    Note: This decision has been made based on prior experience seeing removal of overuse of `<nav>` elements from www.GOV.UK, which made it confusing for users of assistive technologies to know what were the most important navigation aspects of GOV.UK.
+    Should be challenged if user research indicates this is an issue.
+
+  - entire component must have a role of `contentinfo` (https://www.w3.org/TR/wai-aria-1.1/#contentinfo)
+    Note: This decision has been made given that most uses of this role tend to be used directly on a `<footer>` element.
+    Assumption made is that is most predictable for users of assistive technologies.
+    The spec indicates that `contentinfo` is useful for "Examples of information included in this region of the page are copyrights and links to privacy statements.", which may indicate that it might be better placed around the 'meta' section of this component.
+    Should be challenged if user research indicates this is an issue.
```

Comment:
I'm sure there's a good explanation for this but I'm not clear why the accessibility criteria are in the YAML file?

👍 for adding them.

Reply:
I imagined eventually we'd want to expose these somewhere, either in the README, Design System or Review application.

For now it's in the yaml.